test/fuzz/createCleartextMessage.js

```diff
@@ -0,0 +1,15 @@
+import { FuzzedDataProvider } from '@jazzer.js/core';
+
+import openpgp from '../initOpenpgp.js';
```

Comment:
Minor, but that entrypoint is just for mocha tests, I think it's best to use `import * as openpgp from 'openpgp';` (or just `import { <method> } from 'openpgp'`) in all these fuzz tests instead, to ensure we test the released bundle as-is 🙂